functions-and-scope.js

```diff
@@ -57,7 +99,8 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 /* 2c: Afronden op twee decimalen */
 // Zorg ervoor dat het gemiddelde cijfer dat wordt teruggegeven uit de functie netjes wordt afgerond op twee decimalen.
 // Tip: Google is your best friend!
-
+let averageTwodecimals = Math.round(100 * averageGrade(grades)) / 100;
```

Comment:
`const averageTwodecimals = averageGrade(grades).toFixed(2);` is wat netter. `toFixed` is ook precies waar het voor bedoeld is

Reply:
Die kon ik niet vinden, daarom had ik miojn eigen ToFixed geschreven :-)